wgpu/src/lib.rs

```diff
@@ -3982,6 +4066,25 @@ impl Surface {
             SurfaceStatus::Lost => return Err(SurfaceError::Lost),
         };
 
+        let config = self
+            .config
+            .lock()
+            .expect("This surface has not been configured yet.");
```

Comment:
What should I do in case the `SurfaceConfiguration` is not available? Does a panic suffice?